pkg/gadgets/trace/bind/types/types.go

```diff
@@ -15,20 +15,25 @@
 package types
 
 import (
+	"github.com/kinvolk/inspektor-gadget/pkg/columns"
 	eventtypes "github.com/kinvolk/inspektor-gadget/pkg/types"
 )
 
 type Event struct {
 	eventtypes.Event
 
-	Pid       uint32 `json:"pid,omitempty"`
-	Comm      string `json:"comm,omitempty"`
-	Protocol  string `json:"proto,omitempty"`
-	Addr      string `json:"addr,omitempty"`
-	Port      uint16 `json:"port,omitempty"`
-	Options   string `json:"opts,omitempty"`
-	Interface string `json:"if,omitempty"`
-	MountNsID uint64 `json:"mountnsid,omitempty"`
+	Pid       uint32 `json:"pid,omitempty" column:"pid,width:7"`
+	Comm      string `json:"comm,omitempty" column:"comm,width:16"`
+	Protocol  string `json:"proto,omitempty" column:"proto,width:6"`
```

Comment:
Possible values to print are `UDP` or `TCP`.  Hence I think we can use fixed there. 

```suggestion
	Protocol  string `json:"proto,omitempty" column:"proto,width:5,fixed"`
```